hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -35,7 +35,9 @@
 (def local-server-jar
   "Relative to server fat jar"
   (str local-server-dir
-       "/target/jepsen.hazelcast-server-0.1.0-SNAPSHOT-standalone.jar"))
+       "/target/jepsen.hazelcast-server-"
+       (System/getProperty "projectname.version")
```

Comment:
Since this is not the same project, their versions aren't necessarily going to be the same. Might be more robust to look at the target directory contents instead.